CHANGELOG_DEV.md

```diff
@@ -8,4 +8,5 @@ HumHub Changelog
 - Enh #4347: Add `hideMembersSidebar` to make members sidebar optional in space stream page
 - Enh #4585: Group notifications must be enabled explicitly
 - Fix #4646: Clean up duplicated array keys
-- Enh #242: Allow multiple default spaces for groups
\ No newline at end of file
+- Enh #242: Allow multiple default spaces for groups
+- PR #48: Improve "Admin Can View All" Feature
```

Comment:
Just write Enh #48 even if its an PR

Reply:
ok fixed here https://github.com/humhub/humhub/pull/4693/commits/262c2fccef361d7a80d07b56e7af5e28c6647a9c#diff-df55e56bfe234313eade0c101b42fc90f37782988f200fc61c311226b9bc4c4bR12, but I used PR because @luke- wrote here https://github.com/humhub/tasks/issues/48#issue-757285010